assets/js/modules/analytics-4/utils/validation.js

```diff
@@ -20,6 +20,7 @@
  * Internal dependencies
  */
 import { PROPERTY_CREATE, WEBDATASTREAM_CREATE } from '../datastore/constants';
+import { isValidAccountID } from '../../tagmanager/util/validation';
```

Comment:
This makes sense to use the same utility from Tag Manager but we generally don't import anything from another module except constants because they're used in separate bundles. Perhaps `isValidNumericID` could be extracted into a common utility?

Reply:
Makes sense. I moved `isValidAccountId` to a common utility.